activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Moved database-specific ActiveModel types into ActiveRecord
+
+    ie. DecimalWithoutScale, Text and UnsignedInteger
```

Comment:
Can we condense this down to `Moved DecimalWithoutScale, Text, and UnsignedInteger from ActiveModel to ActiveRecord`